src/main/webapp/js/map.js

```diff
@@ -0,0 +1,42 @@
+      let map;
```

Comment:
entire file looks like its indented too much

Reply:
Hi Emily, I just changed the format of the map.js file.